doc/nrf/gs_programming.rst

```diff
@@ -41,7 +41,7 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <g
 
 1.    Open a terminal window.
```

Comment:
Line 40: Shouldn't we update the link :ref:`automatic <gs_app_installing-ncs-tcm>` to point to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8771/nrf/gs_assistant.html#installing-command-line-requirements?